config/vufind/config.ini

```
@@ -1449,6 +1449,18 @@ replace_other_urls = true
 ; to false.
 ;prefixLinks = true
 
+; Use a web service to determine whether to prefix each link, ignoring prefixLinks
+; above.  Query the configured URL via HTTP GET and a single query parameter, 'url',
+; set to the link domain. The web service should return a body with a single number,
+; '1' to prefix and '0' if not.  The web service should be hosted closely such that
+; query time is minimal.
+; See https://github.com/lehigh-university-libraries/ezproxy-url-checker for an OSS
+; implementation of this protocol that extrats the answers from EZproxy config.
+;prefixLinksWebServiceUrl = http://localhost:8080/proxyUrl
```

```suggestion
;prefixLinksWebServiceUrl = http://localhost:8888/proxyUrl
```

I changed the default port for that service to use `8888` since `8080` was being used by something else. May want that default set here, too.